### PR TITLE
24.0.1 release

### PR DESCRIPTION
**Pull request by pferraro**

### CODE REVIEW

File: _data/releases.yaml [hunk 1]
```diff
@@ -1,3 +1,68 @@
+- version: 24.0.1.Final
+  version_shortname: 24.0.1
+  date: 2021-07-27
+  link:
+    - name: WildFly Preview EE 9 Distribution
```
Comment (bstansberry) [inline _data/releases.yaml:5]: EE 9.1

File: _data/releases.yaml [hunk 2]
```diff
@@ -1,3 +1,68 @@
...
+          size: 209 MB
+          url: https://download.jboss.org/wildfly/24.0.1.Final/wildfly-preview-24.0.1.Final.zip
+          checksum: SHA-1
+          checksum_url: https://download.jboss.org/wildfly/24.0.1.Final/wildfly-preview-24.0.1.Final.zip.sha1
+        - format: tgz
+          size: 208 MB
+          url: https://download.jboss.org/wildfly/24.0.1.Final/wildfly-preview-24.0.1.Final.tar.gz
+          checksum: SHA-1
+          checksum_url: https://download.jboss.org/wildfly/24.0.1.Final/wildfly-preview-24.0.1.Final.tar.gz.sha1
+    - name: Jakarta EE Full & Web Distribution
```
Comment (bstansberry) [inline _data/releases.yaml:18]: EE 8

**Review by bstansberry — CHANGES_REQUESTED** [review]

File: _data/releases.yaml (hunk 2 above)
Comment (pferraro) [reply _data/releases.yaml]: Bon point.

File: _data/releases.yaml (hunk 1 above)
Comment (pferraro) [reply _data/releases.yaml]: Bon point.

**Pull request merged**